src/main.py

```diff
@@ -41,7 +41,7 @@
 @click.option("--input_data", required=True, default= '../data/utopia.txt', 
               help="The path to the input datafile. relative from the src folder, e.g. '../data/utopia.txt'")
 @click.option("--solver", default="cbc", 
-              help="If another solver than 'glpk' is desired please indicate the solver. [gurobi]")
+              help="If another solver than 'cbc' is desired please indicate the solver. [gurobi]")
```

Comment:
In the brackets it should then also list glpk, is gurobi now implemented?

Reply:
Good catch. Revised wording